CONTRIBUTING.md

```diff
@@ -32,7 +32,15 @@ worked.
 
 ##### Building the documentation
 
+The MNE-BIDS documentation is hosted with GitHub pages from the `gh-pages` branch.
+
 The documentation can be built using sphinx. For that, please additionally
 install the following:
 
     $ pip install sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme pillow
+
+To build the documentation, run `make html` from inside the `doc/` directory. You
+can then preview the documentation from the `_build/html/` directory.
+
+The documentation is automatically deployed to `gh-pages` when the `master` branch is
+updated using CircleCI (`.circleci/config.yml`).
```

Comment:
this is no longer true actually. Only the `maint/0.1` branch is updated. The reason is that we don't want to have a version mismatch between the pip installed version and the one on github.io.

Reply:
isn't the whole point of developer documentation to cover all the things one needs to know in order to maintain the package? CI/CD seem like a core part of that

Reply:
well, I would differ on this. I think the point of developer documentation is to make it as easy as possible to make a contribution to the package -- not necessarily to maintain it. 

We don't have a big maintenance team, so trying to document all of this would mean that developers take longer to go through information that's actually relevant for contributing. In the life cycle of making a pull request, a developer would rarely if ever need to know how to deploy the documentation to github.

Reply:
My issue is with the second sentence, not the first. The first is okay -- and it is indeed useful !